config/settings/base.py

```diff
@@ -509,6 +509,45 @@
     default=f"https://{FILE_UPLOAD_BUCKET}.s3.amazonaws.com",
 )
 
+ALLOWED_MIME_TYPES = env.json(
+    "ALLOWED_MIME_TYPES",
+    default=[
```

Comment:
Either switch this to a `list` or a `tuple` method from `env` or make the default value a json object.